content/developers/metrics/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Metrics
+kind: documentation
+js_dd_docs_methods:
+  - metricsGuidePage
+code_languages:
+  - Python
+  - Ruby
+aliases:
+  - /metrics/
+  - /guides/metrics/
+  - /metrictypes/
+  - /units/
+further_reading:
+- link: "developers/dogstatsd"
+  tag: "Documentation"
+  text: Learn more about DogStatsD
+- link: "developers/libraries"
+  tag: "Documentation"
+  text: Official and Community-contributed API and DogStatsD client libraries
+---
+
+## Introduction
+
+This section explains the nuts and bolts of metrics - what they are, and what they do. Whether you want to send [custom metrics][4], or just want to have a better understanding about how Datadog works, read on. If you're looking for information about the DogStatsD (which implements these metrics), please see the [DogStatsD documentation][5].
+
+### Submitting metrics
+
+There are multiple ways to send metrics to Datadog:
+
+1. Via the Datadog Agent directly. Learn how [to write an Agent check](/agent/agent_checks), or examine the [Aggregator source code][9] directly.
+2. Via the DogStatsD server (bundled with the Datadog Agent) and a [client library][16].
+3. Directly via Datadog's [HTTP API][10].
+4. Via Dropwizard's Java [metrics][11] library with the [metrics-datadog][12] backend. Thanks to the good folks at [Vistar Media][19], [Coursera][13], and [Bazaarvoice][14] for their contributions.
+
+### Naming metrics
+
+There are a few rules regarding metric names:
+
+* Must start with a letter.
+* Must only contain ASCII alphanumerics, underscores, and periods.
+  * Other characters, including spaces, are converted to underscores.
+  * Unicode is _not_ supported.
+* Must not exceed 200 characters. Fewer than 100 is preferred from a UI perspective.
+
+Metrics reported by the Agent are in a pseudo-hierarchical dotted format (e.g. `http.nginx.response_time`). The hierarchy is neither enforced nor interpreted, but it can be used to infer things about servers (e.g. "hey, I see hostA and hostB are reporting `http.nginx.*`, those must be web frontends").
+
+## Metric Types
+
+The "Datadog in-app type" affects how a given metric is interpreted in query results and graph visualizations across the application. The metric type visible on the metric summary page is the Datadog in-app type. You should only change the type if you have started submitting this metric with a new type, and should be aware that changing the type may render historical data nonsensical.
+
+In the Datadog web application there are four metric types (though one is deprecated):
+
+* GAUGE
+* RATE
+* COUNT
+* COUNTER (deprecated)
+
+A metric's type is stored as metrics metadata and is used to determine how a metric is interpreted throughout the application by determining default time aggregation function and `as_rate()`/`as_count()` behavior. The `as_count()` and `as_rate()` modifiers behave differently for different Web Application metric types.
```

Comment:
Should we link to each one of the dedicated metrics type section `In-app modifiers`?

Reply:
Imho no - it would just be a mess of links.

Reply:
I'm not sure. Frankly this whole section needs some love, and I strongly suspect that _somebody_ will be re-writing it in the future. 😉 `wontfix` for now.